07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,61 @@
+body{
+  width:60%;
+  margin:auto;
+  font-family:sans-serif;
+}
+.bgimg{
+  background-image:url('../img/nasa-background.jpg');
+  height:450px;
+  width:850px;
+  background-position: center;
+}
+.nasa{
+  color:white;
+  text-align: center;
+  position:relative;
+  margin-top:-275px;
+  font-family:arial;
+  margin-left:40px;
+  font-size:80px;
+}
+
+h3{
+  font-size:20px;
+}
+.text{
+  background-color:#e0e0e0;
+  margin-top:220px;
```

Comment:
You could take the margin-top down a bit here, but overall this looks solid. Good job!